tensorflow/inference/docker/2.12/py3/Dockerfile.ec2.cpu.py_scan_allowlist.json

```diff
@@ -0,0 +1,3 @@
+{
+    "12345": "Custom Ignore message"
+}
```

Comment:
Does this need to be removed before merge?

Reply:
Yes, this needs to be removed, thanks!